ui/src/app/views/concept-set-list/component.html

```diff
@@ -43,5 +43,10 @@
     </div>
   </div>
 </app-top-box>
+<app-create-concept-modal *ngIf="isCreateModalOpen"
+                          [onCreate]="loadConceptSets.bind(this)"
```

Comment:
We should bind these once in the constructor, rather than inline here (perf reasons).